docs/content/preview/yugabyte-cloud/cloud-secure-clusters/cloud-users.md

```diff
@@ -56,10 +60,16 @@ Although not a superuser, `yb_superuser` includes sufficient privileges to perfo
 
 - Has the following role options: `INHERIT`, `CREATEROLE`, `CREATEDB`, and `BYPASSRLS`.
 
-- Member of the following roles: `pg_read_all_stats`, `pg_signal_backend`, and [yb_extension](#yb-extension).
+- Member of the following roles: `pg_read_all_stats`, `pg_signal_backend`, `yb_db_admin`, and [yb_extension](#yb-extension).
 
 `yb_superuser` is the highest privileged role you have access to in YugabyteDB Managed. You can't delete, change the passwords, or login using the `postgres` or `yugabyte` superuser roles.
 
+### yb_db_admin
+
+The `yb_db_admin` role is specific to YugabyteDB Managed clusters and allows non-superuser roles to perform a number of privileged operations in a manner similar to a superuser, such as performing operations on objects it does not own. A user granted this role can only alter the BYPASSRLS attribute of a role.
```

Comment:
"A user granted this role can only alter the BYPASSRLS attribute of a role."

Perhaps we can leave this sentence out for now. We'll add on more role properties yb_db_admin can edit, but it's a work in progress. I'll follow up with you when we have all the role editing properties sorted out (along with updating the functional doc for yb_db_admin).